docs/HACKING.md

```diff
@@ -291,3 +291,41 @@ the `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` environment variable like so:
 ```
 OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/user/origin-release:v{version number} bin/openshift-install create cluster --log-level=debug
 ```
+
+## Hacking on `machine-os-content`
+
+If you own part of the operating system (from kernel to kubelet) you
+are part of the `machine-os-content`.  More information in [OSUpgrades.md].
+You will want a workflow for testing changes to a cluster.
+
+By far the simplest in a lot of cases is going to be to use `oc debug node/`
+to start by testing your changes on e.g. a single worker node.
+Commands to use here are e.g. `rpm-ostree usroverlay` and/or `rpm-ostree override replace`.
+The first one gives you a writable overlayfs on `/usr` and you can go to town
+replacing binaries there (e.g. `/usr/bin/crio`) or whatever.  If you want to install
+a test kernel in particular you'll need to use `rpm-ostree override replace`.
+
+A much more advanced (but "correct") flow is to build a custom `machine-os-content`
```

Comment:
"A more advanced flow is to build a custom `machine-os-content` container" Suggested extra sentence to explain why this choice is a good one?